pkg/report/record.go

```diff
@@ -32,6 +32,23 @@ type Record struct {
 	MasterVersion *string `json:"masterVersion,omitempty"`
 	// Nodes is a list of node-specific information from the reporting cluster.
 	Nodes []Node `json:"nodes,omitempty"`
+	// Namespace contains statistics regarding the k8s namespace objects
+	Namespace Namespace `json:"namespaces,omitempty"`
+}
+
+type Namespace struct {
```

Comment:
Call this NamespaceStats - it doesn't represent a Namespace like a Node struct does.